orchestrator-salt/src/main/resources/salt-common/salt/monitoring/init.sls

```diff
@@ -21,4 +21,20 @@ start_monitoring:
 {% endif %}
 
 {% endif %}
+/opt/salt/scripts/cdp_resources_check.sh:
+  file.managed:
+    - makedirs: True
+    - user: root
+    - group: root
+    - mode: 700
+    - source: salt://monitoring/scripts/cdp_resources_check.sh
```

Comment:
if this is a jinja, then probably it is worth renaming to .sh.j2

Reply:
actually it's not so i will remove the "template: jinja" part